### ISSUE TITLE
Update docs to include log_level and line-color

### ISSUE THREAD

**Issue by vajahath** [vajahath/lme#24]
## Opening discussion on documentation structure

If we add docs for LOG_LEVEL and line-color support (and docs of future changes) to the current single page readme, it may go big and become unreadable. So we need to think about opening Wiki for this.

### What I'm proposing?
Let's split the documentation to 4 parts.

- basic functionality docs (added to readme)
- a very good index which maps to Wiki's pages (added to both readme and Wiki)
- advanced functionality docs (added to Wiki)
- Contributors and guide to them, thanks, licence.

Next we've to determine what should be included in them
- basic functionality
  - basic logging feature:  `lme.*(args [, args])`
  - basic line-drawing feature: `lme.*line([char, len])`
- advanced functionality
  - log-level feature
  - custom log-color feature
  - custom line-color feature

**Comment by vajahath** [comment]
addressing this..

**Comment by vajahath** [comment]
@demacdonald can you find some time adding documentation to log_level support at the [wiki/Adjust-Logging-Level](https://github.com/vajahath/lme/wiki/Adjust-Logging-Level) ?

**Comment by demacdonald** [comment]
Sure, I'll find time within the next couple days, no problem.

**Comment by vajahath** [comment]
@demacdonald hi 🎈

**Comment by demacdonald** [comment]
Yeah, sorry... work has been a little insane... I'll try to get to it tonight :(

**Comment by vajahath** [comment]
No actually I was just trying to add a reminder in case if you've forgotten.

**Comment by demacdonald** [comment]
Okay, I have some basic documentation in place. Let me know your thoughts...might be easier to talk here: https://gitter.im/lme-logger/Lobby?utm_source=share-link&utm_medium=link&utm_campaign=share-link instead of making issues get messy and hard to follow

**Comment by vajahath** [comment]
Great! :+1:
